modules/configure-signature-integration.adoc

```diff
@@ -17,8 +23,28 @@ Before performing image signature verification, you must first add your Cosign p
 . Scroll down to the *Signature Integrations* section and click *Signature*.
 . Click *New integration*.
 . Enter a name for the *Integration name*.
-. Click *Cosign* → *Add a new public key*.
+. Click *Cosign public Keys* → *Add a new public key*.
 . Enter the *Public key* name.
 . For the *Public key value* field, enter the PEM-encoded public key.
 . (Optional) You can add more than one key by clicking *Add a new public key* and entering the details.
 . Click *Save*.
+
+== Configuring Cosing Certificates
+
+.Prerequisites
+
+* You must already have the certificate identity, and issuer. Optionally, you also need a PEM-encoded certificate and chain. For more information about Cosign Certificates, see link:https://docs.sigstore.dev/verifying/verify/#verify-image-with-user-provided-trusted-chain[Cosign Certificates]
+
+.Procedure
+
+. In the {product-title-short} portal, select *Platform Configuration* → *Integrations*.
```

Comment:
```suggestion
. In the {product-title-short} portal, select *Platform Configuration* -> *Integrations*.
```
We need to use `->` so that it transforms properly in our doc publishing.